packages/components/src/interfaces/complex/Contentpic.ts

```diff
@@ -0,0 +1,30 @@
+import type { Static } from "@sinclair/typebox"
+import { Type } from "@sinclair/typebox"
+
+import { BaseProseSchema } from "../native/Prose"
+
+export const ContentpicSchema = Type.Object(
```

Comment:
issue: the render engine is missing information about this component so it won't be able to render. Need to add this component to:
- [renderComponent under render.tsx](https://github.com/opengovsg/isomer/blob/fa7b0b31379a3426ffa41a5da0f40b97142decd8/packages/components/src/templates/next/render/render.tsx#L32)
- [IsomerComplexComponentsMap under components.ts](https://github.com/opengovsg/isomer/blob/fa7b0b31379a3426ffa41a5da0f40b97142decd8/packages/components/src/schemas/components.ts#L22)
- [DEFAULT_BLOCKS under constants.ts](https://github.com/opengovsg/isomer/blob/fa7b0b31379a3426ffa41a5da0f40b97142decd8/apps/studio/src/components/PageEditor/constants.ts#L4)

If ready can add to [ComponentSelector.tsx](https://github.com/opengovsg/isomer/blob/57d1f2f6c75a9291e49210dba5244244bfded8c4/apps/studio/src/components/PageEditor/ComponentSelector.tsx) as well (although need to know the layout of the page beforehand).